### ISSUE TITLE
InvalidTypeHint when used typehint for first parameter as only typehint

### ISSUE THREAD

**Issue by CZechBoY** [slevomat/coding-standard#407]
Hi,
I have this method
```php
    /**
     * @param object $a
     * @param object $b
     */
    public function __construct(string $firstParameter, $a, $b)
    {
    }
```
and this is the output
```
 29 | ERROR | Unknown type hint "string" found for $a
    |       | (Squiz.Commenting.FunctionComment.InvalidTypeHint)
 91 | ERROR | Unknown type hint "string" found for $value
    |       | (Squiz.Commenting.FunctionComment.InvalidTypeHint)
```
I would expect, that this code will pass.
When I add "@param string $firstParameter" to phpdoc, it does not complain anymore. Also when I add typehints to "object parameters" and remove phpdoc completely it is without warning. When I have object typehints and phpdoc from above it outputs warnings too.

Version: 4.6.3
PHP: 7.1

**Comment by CZechBoY** [comment]
I think I am at bad repo, sorry :-)

**Comment by kukulich** [comment]
See the error codes. There don’t come from the Slevomat sniffs.
